### DISCUSSION TITLE
Automated writers name and date info.

### DISCUSSION THREAD

**Discussion by dusohee** [GitbookIO/community#261] [Feature Requests]
Recently, I started to use gitbook at work and we made space for comapany's documents so now, over 80 people using it. 
There are many writers with differents id, but there is no functions to show writers info and date. 

The date and writer info help to mange documents like how latest, who wrote.. 
If it's outdated, we need to ask to update contents SO I suggest that Gitbook publishes date and writers info automatically when documents merged or edit. 

I hope many suggestions make this platform better. 
Thank you

**Comment by taszap** [comment]
Hey @dusohee 

Unless I am missing some context you can already do this by visiting the change history tab ([see here](https://docs.gitbook.com/content-creation/activity-history#viewing-historical-versions-of-content))

The feed shows you recent changes, date and who has made them. 
<img width="396" alt="image" src="https://github.com/GitbookIO/community/assets/63643069/3078b76e-cf37-4d49-b26f-ded703dcc943">

**Comment by ShutdownRepo** [comment]
Would it be possible to show basic info like authors and latest edit date on the top or bottom of the page ? It’s kinda already there on edit mode but not on published documentation, which prevents unauthenticated users from finding out who contributed to the page and when was the page last edited

**Reply by taszap** [reply]
Hey @ShutdownRepo we've intentionally removed it on public docs. 
Other users were concerned that in some cases documentation was updated long time ago but this didn't make it invalid (for example specific part of the documentation hasn't changed much). Not everyone wanted to display authors and their profile images/icons publicly either for privacy reasons etc. 

I would say it's unlikely we would bring it back on public docs (but of course, never say never if this becomes a frequent feedback)!

**Reply by ShutdownRepo** [reply]
Understood
Is there any way we could access that data in order to develop a custom integration that allows to fit our needs? To date, if we want to highlight a doc's authors, there's no way we can achieve this besides doing it manually, which is not feasible when there are many pages

**Reply by taszap** [reply]
Hey @ShutdownRepo
Looping in my colleague @addisonschultz who works a lot with our integrations

**Reply by addisonschultz** [reply]
Hey @ShutdownRepo, definitely a nice use-case! I think this should be possible to build, using GitBook's API. 

We have this endpoint that would give you information about all the Change Requests made in a space: https://developer.gitbook.com/gitbook-api/reference/spaces/change-requests#get-a-list-of-change-requests-in-a-space

It tells you the information you're looking for - like the date and author!

**Reply by ShutdownRepo** [reply]
There is a misunderstanding. The `change-requests` endpoint gives information regarding edits that were marked "request for review". 
What I'm looking for is how to get the authors of an existing page (circled in yellow below).
![image](https://github.com/GitbookIO/community/assets/40902872/745dd7d1-8ff3-45cc-8193-47daf97ffd1b)
The `content/path` endpoint already allows to get the content of a page and many information associated, like its title, id, kind, type, description, ... but not the authors. That's what I'm looking for.
This would allow to dev an integration that pulls pages' authors, and then highlight them.

**Reply by addisonschultz** [reply]
I see - Unfortunately I don't think we have an open endpoint for that specifically. The one I posted above could give you a "rough" picture of people creating the change requests, but not everyone who's edited it. 

I'll make a note for this so we could look into it more. 

On this note though, I'm wondering if an integration could be built for this. We have an event to tell when a space is updated: https://developer.gitbook.com/integrations/reference/event#space_content_updated - and I believe the payload should contain information about the user who's made the update. 

I do think though, that this only is fired, upon CRs when they merge - and might possibly contain only information about the user who's merged it.

**Reply by ShutdownRepo** [reply]
I have no knowledge of how gitbook works internally, but wouldn't it be simple/possible to return the authors in the content/path endpoint ? It would make sense 100%

**Reply by addisonschultz** [reply]
It can be a little more complicated than that - but I've made sure to log this in our feature requests for us to visit in the (hopefully) near future!
